### DISCUSSION TITLE
Reconsider badges color scheme

### DISCUSSION THREAD

**Discussion by hassila** [SwiftPackageIndex/SwiftPackageIndex-Server#3410] [Ideas]
I got a comment on the colouring of the badges that is generated from SPI for the Benchmark package - basically it was thought it was an error/warning colour and that there was a problem (see https://github.com/ordo-one/package-benchmark/issues/205#issuecomment-2383826967) .

Maybe it is worth using the Swift logo color to the left of the badge as is, but have some green color for "ok" for the supported compiler versions/os versions, as it is a positive things that those are supported (and now it was construed as that there was a problem with the CI pipeline for those versions...).

**Comment by hassila** [comment]
<img width="896" alt="image" src="https://github.com/user-attachments/assets/1bb18ebe-2393-456c-ab98-5caacd8be06d">

Attaching screenshot - the orange here was viewed as "red".

**Comment by hassila** [comment]
(and codecov also uses orange as warning color due to not enough test coverage)

**Comment by hassila** [comment]
So the rationale for change would be that most people expect badges to signal how the health of some part of the CI pipeline is, while the SPI one is using the Swift logo color.

**Comment by daveverwer** [comment]
I can see how people would think this, however I'd want to be cautious using green for this badge.

You already mentioned that people see these as “most people expect badges to signal how the health of some part of the CI pipeline is” and we are very much not trying to be CI for every open-source Swift package.

If we change this colour, we should move to a blue or grey or other less “loaded” colour that shows information rather than being seen as indicating passing/failing.

**Comment by hassila** [comment]
That makes a lot of sense, agree - some neutral color would be great.

**Comment by daveverwer** [comment]
Tracked as #3411
